packages/vue/src/components/organisms/SfProductCardHorizontal/SfProductCardHorizontal.vue

```diff
@@ -93,10 +93,10 @@
           <!--@slot Use this slot to replace add to cart-->
           <slot name="add-to-cart">
             <SfAddToCart
-              :qty="qty"
+              v-model="quantity"
```

Comment:
Can't we just use `this.qty`? 
```suggestion
              v-model="qty"
```